apps/playground/src/stories/hooks/useHover.stories.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useCallback, useMemo, useRef, useState } from 'react';
+import { useHover } from '../../../../../packages/react-hooks/use-hover';
+// import { useHover } from "@nanlabs/react-hooks";
+
+export const Example = () => {
+  const [isCbMemoBroken, setIsCbMemoBroken] = useState(false);
+  const [isTargetMemoBroken, setTargetMemoBroken] = useState(false);
+  const [events, setEvents] = useState<string[]>([])
+
+  const getEventHandler = useCallback((eventName: string) => {
+    return () => setEvents((events) => [...events, `"${eventName}" callback fired`])
+  }, [setEvents])
+
+  const memoizedCallbacks = useMemo(
+    () => ({
+      onChange: getEventHandler('onChange'),
+      onLeave: getEventHandler('onLeave'),
+      onEnter: getEventHandler('onEnter'),
+    }),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [getEventHandler, isCbMemoBroken]
+  )
+
+  const notMemoizedCallbacks = {
+    onChange: getEventHandler('onChange'),
+    onLeave: getEventHandler('onLeave'),
+    onEnter: getEventHandler('onEnter'),
+  }
```

Comment:
@boredcity applying the suggestions made by @coderabbitai will be great!

Reply:
I don't think it makes sense to copy `eventHandlers` to `notMemoizedCallbacks` (it would actually make code even less optimal) but as long as everyone is on board with useMemo hook using `eventHandlers` but relying on `getEventHandler`, `isCbMemoBroken` I'm ok to make the change :)